Colors plugin has incorrect TS type

### Expected behavior

The official documentation indicates that the Colors plugin should be register like this (https://www.chartjs.org/docs/latest/general/colors.html#default-color-palette):

```ts
import { Colors } from 'chart.js';

Chart.register(Colors);
```

### Current behavior

However, this does not compile in TS (see the TypeScript playground below):

```
error TS2345: Argument of type '{ id: string; defaults: ColorsPluginOptions; beforeLayout(chart: Chart<keyof ChartTypeRegistry, (number | [number, number] | Point | BubbleDataPoint | null)[], unknown>, _args: any, options: ColorsPluginOptions): void; }' is not assignable to parameter of type 'ChartComponentLike'.
  Type '{ id: string; defaults: ColorsPluginOptions; beforeLayout(chart: Chart<keyof ChartTypeRegistry, (number | [number, number] | Point | BubbleDataPoint | null)[], unknown>, _args: any, options: ColorsPluginOptions): void; }' is not assignable to type 'Plugin<keyof ChartTypeRegistry, AnyObject>'.
    Types of property 'defaults' are incompatible.
      Type 'ColorsPluginOptions' is not assignable to type 'Partial<AnyObject>'.
        Index signature for type 'string' is missing in type 'ColorsPluginOptions'.
```

### Reproducible sample

https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgYQBYENYBoUQDbQDOOAMsAHYCmyE5MU+elUcAvnAGYMhwDkAxhlgA6AFaFeAbgBQ0tJhjColAObBCMZgAoyVGnQZ4mUAJQyA9ObgATCJUJxyEePwjhgTOUMXK1G7TQEUIRmQA

### Optional extra steps/info to reproduce

_No response_

### Possible solution

`Chart.register(Colors as any)`

### Context

_No response_

### chart.js version

v4.1.1

### Browser name and version

_No response_

### Link to your project

_No response_

--- a/src/types/basic.d.ts
+++ b/src/types/basic.d.ts
@@ -1,3 +1,3 @@
 
-export type AnyObject = Record<string, unknown>;
+export type AnyObject = Record<string, any>;
 export type EmptyObject = Record<string, never>;

--- a/test/types/register.ts
+++ b/test/types/register.ts
@@ -23,7 +23,8 @@ import {
   Legend,
   Title,
   SubTitle,
-  Tooltip
+  Tooltip,
+  Colors
 } from '../../src/types.js';
 
 Chart.register(
@@ -50,5 +51,6 @@ Chart.register(
   Legend,
   Title,
   SubTitle,
-  Tooltip
+  Tooltip,
+  Colors
 );

